### PR TITLE
Fix tab order

### PR DESCRIPTION
**Pull request by orangejenny**
The edit buttons for display/validation conditions, and the "auto?" checkboxes for various advanced properties, grab focus before their text boxes, which is annoying if you're tabbing through properties. It's easier to change the markup & styling than to futz with global tabindex.

Checked that this looks & behaves the same in Chrome, Firefox, Safari.

### CODE REVIEW

File: src/javaRosa.js [hunk 1]
```diff
@@ -464,15 +464,16 @@ define([
         var _getUIElement = widget.getUIElement;
         widget.getUIElement = function () {
             var $uiElem = _getUIElement(),
-                $autoBoxContainer = $('<div />').addClass('pull-right fd-itextID-checkbox-container'),
+                $autoBoxContainer = $('<div />').addClass('fd-itextID-checkbox-container'),
                 $autoBoxLabel = $("<label />").text("auto?").addClass('checkbox');
 
             $autoBoxLabel.prepend($autoBox);
             $autoBoxContainer.append($autoBoxLabel);
+            $uiElem.css('position', 'relative');
```
Comment (orangejenny) [inline src/javaRosa.js]: I don't love having styling in js files, but making .widget.control-group position: relative in LESS affects the layout of the basic fields.

**Comment by orangejenny** [comment]
@nickpell

**Comment by nickpell** [comment]
notifying @millerdev and @emord

**Comment by millerdev** [comment]
I've been annoyed with in the past, but never got around to fixing it. Nice work!

**Pull request merged**